src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -83,9 +145,23 @@ export function transformVAOSAppointment(appt) {
     },
     preferredTimesForPhoneCall: appt.preferredTimesForPhoneCall,
     comment: appt.comment,
-    // TODO missing full video data
+    // TODO missing video data: status
```

Comment:
Hmm, not sure if we actually need a status, since I think we pretty much ignore it for video appointments. I guess we'll wait and see if it becomes necessary.